docs/Learning-Environment-Design-Agents.md

```diff
@@ -553,8 +542,8 @@ up to use either the continuous or the discrete vector action spaces.
 ### Continuous Action Space
 
 When an Agent uses a Policy set to the **Continuous** vector action space, the
```

Comment:
This is the part that I wasn't sure how to rewrite until the BrainParameters refactor is in. Can revisit then.

Reply:
maybe we can reword it as "When an Agent's Policy has a **Continuous** vector action space, ..."

Reply:
That's better. I think I'll go with "When an Agent's Policy has **continuous** actions..." (trying to get away from "space")